corehq/apps/analytics/tasks.py

```diff
@@ -368,7 +369,8 @@ def track_periodic_data():
     submit = []
     for user in users_to_domains:
         email = user.get('email')
-        if not email:
+        if not email or not validate_email(email, verify=True):
```

Comment:
i think it still gets run (unless there is something excluding it from being picked up as a task), but you're right i doubt anything works so it's probably not a big deal